README.md

```diff
@@ -1,2 +1,51 @@
 # NGI-CellRaiser
 Pipeline for analysing 10X single cell data
+
+**This is work in progress.** We actually want to extend the pipeline collection for scRNAseq, contribute to the FAIR guiding principles Wilkinson et al. 2016 (doi:10.1038/sdata.2016.18) and share.
+
+If you want to contribute, please write an email to alexander.peltzer@qbic.uni-tuebingen.de or sven.fillinger@qbic.uni-tuebingen.de.
+
+## Notes
+
+There already exists a quality assesment pipeline, https://github.com/Teichlab/celloline, but it is only executable on an amazon cluster. **I think that we can learn a lot from what they implemented there!**
+
+## Pipeline design ideas
+
+The core upstream module of the pipeline shall be https://github.com/CGATOxford/UMI-tools.
+
+Here is a hypothetical upstream workflow:
+
+| Step | Tool | Inputs | Output |
+| -------- | -------- | -------- | -------- |
+| Quality Control of FASTQs | FASTQC | R1 + R2 FASTQs | HTML Report
+| Quality Control - Detect Outliers (Gini Index) | http://www.morgridge.net/SinQC.html | R1 FASTQ | ugly report (PDFs + Tables)
+| Find Cell barcode whitelist (OPTIONAL) | umi_tools whitelist | R1 FASTQ | whitelist.txt |
+| Extract CB/UMIs and filter CBs | umi_tools extract | R1 + R2 FASTQs + whitelist.txt | extracted FASTQs |
+| Remove PCR duplicates | umi_tools dedup | | |
+| Map reads | STAR, Kallisto, Salmon | extracted FASTQs | BAM |
+| Assign reads to genes | featureCounts, Kallisto? | BAM + transcriptome GTF | BAM
+| Count unique reads per genes per cell | umi_tools count | BAM | Counts.txt
+
+We have to add a http://multiqc.info/ report for every step where it is necessary.
```

Comment:
NB: To confirm, there should be a single MultiQC process which runs at the end, not one for every step. If there are optional processes then dummy channels have to be created and closed for any step that does not run. Bit of a faff, but I haven't figured out a better system yet.

Reply:
When writing this first draft I had something like that in mind. What I meant with report for every step are several subreports (e.g. FASTQC, STAR, etc.) generated by one MultiQC instance. 
But you are the expert here ;)

Reply:
So I've seen "subreports" like this in one or two other pipelines (eg. our own [CAW](https://github.com/SciLifeLab/CAW)). It works fine from a technical perspective, but I'm not sure how useful they are. In practice, I'm not sure that any of us have ever looked at them before the pipeline has finished. And once it's finished, it's much better to have a single report that has everything in it.

Reply:
I agree with Phil, subreports do not make really sense. At the end of the day you want to have one single report per run.